### PR TITLE
Create Step 2

### PR DESCRIPTION
**Pull request by ConnorKirk**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

This PR creates Step 2 of the WAF workshop

### CODE REVIEW

File: docs/step-2.md [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+* Navigate to the Amazon Athena console, select the Saved Queries tab;
+* Select the query you want to customize (ScannersProbesLogParser);
+
+![athena-saved-queries](2-01-athena-saved-queries.png)
+
+* Apply your changes (ex: change from COUNT(*)>=50 to COUNT(*)>=500);
+* Save the new Athena Query and copy the new query ID.
+
+
+> NOTE: The query ID is part of the URL when you access the saved query (https://console.aws.amazon.com/athena/home?force&region=us-east-1#query/saved/**_query_id_**). Alternatively, you can use the `get-named-query CLI command`;
```
Comment (matteofigus) [inline docs/step-2.md]: I think we can make this url region agnostic for international customers
```suggestion
> NOTE: The query ID is part of the URL when you access the saved query (https://console.aws.amazon.com/athena/home?force#query/saved/**_query_id_**). Alternatively, you can use the `get-named-query CLI command`;
```

File: docs/step-2.md [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+![http-flood-config](2-00-http-flood-config.png)
+
+
+### 2.1.3 Testing the new rules
+Let's test your HTTP flood protection. We will use [Apache AB](https://httpd.apache.org/docs/2.4/programs/ab.html).
+
+> ⚠️ **Warning**: Do not run the benchmarking tool from your local machine!
+
+We will use Systems Manager Session Manager to connect to the instance and run the `ab` benchmarking tool.
```
Comment (matteofigus) [inline docs/step-2.md]: What about adding a link?
```suggestion
We will use [Systems Manager Session Manager](https://console.aws.amazon.com/systems-manager/session-manager/start-session) to connect to the instance and run the `ab` benchmarking tool.
```

File: docs/step-2.md (hunk 1 above)
Comment (ConnorKirk) [reply docs/step-2.md]: Great suggestion. Will do

File: docs/step-2.md [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+Now we will customise our Scanner and Probe rules. These use Amazon Athena.
+The solutions refer to the Athena by a saved query ID. As Athena don't allow you to change saved queries, the process to apply customizations to Athena query is by creating a new query and updating the Athena log parser event to use the new query ID.
+
+### 2.2.1
+* Navigate to the Amazon Athena console, select the Saved Queries tab;
+* Select the query you want to customize (ScannersProbesLogParser);
+
+![athena-saved-queries](2-01-athena-saved-queries.png)
+
+* Apply your changes (ex: change from COUNT(*)>=50 to COUNT(*)>=500);
```
Comment (ConnorKirk) [inline docs/step-2.md]:
```suggestion
* Apply your changes (ex: change from `COUNT(*)>=50` to `COUNT(*)>=500`);
```

**Pull request merged**